src/components/services-view/services-view.component.ts

```diff
@@ -2,8 +2,8 @@ import {ServicesViewController} from './services-view.controller';
 
 export const servicesView: angular.IComponentOptions = {
   bindings: {
-    services: '<',
-    categories: '<'
+    serviceClasses: '<',
+    imageStreams: "<"
```

Comment:
Nit: let's consistently use either single quote or double quote styles in the same declaration